airflow/config_templates/config.yml

```diff
@@ -2268,6 +2268,15 @@ scheduler:
       type: string
       example: ~
       default: "5"
+    local_task_job_heartbeat_sec:
+      description: |
+        The frequency (in seconds) at which the LocalTaskJob should send heartbeat signals to the
+        scheduler to notify it's still alive. The value of this parameter should be less than the 
+        scheduler's `scheduler_zombie_task_threshold` config.
+      version_added: ~
```

Comment:
I think we also need to mention a version since we're are newly adding this param. Like in https://github.com/apache/airflow/pull/32313/files#diff-0a480a8b563e200f749424d7e761555e543588ff39a0b11cce7e34d522f38e0eR2212

And not sure if this needs to be in 2.6.3 or in 2.7.0 release. 

@uranusjr will you please be able to help here with respect to which version could this go in?

Reply:
This isn’t fixing an urgent issue so 2.7.0 probably should be preferred, unless there are specific reasons.

Reply:
agree.

Reply:
@pankajkoti @uranusjr @potiuk Updated the version now